spec/ttml1.xml

```diff
@@ -4598,12 +4598,19 @@ as a style property only to those element types indicated in the following table
 computed value of the style property must be considered to be no less than the
 size of the largest font size that applies to the element and its
 descendant elements in the intermediate synchronic document as determined by
-<specref ref="semantics-region-layout-step-1"/>. However, the value <code>normal</code> is not relative,
-and is therefore inherited as-is, as specified at <specref ref="semantics-style-resolution-processing-css"/>.</p>
+<specref ref="semantics-region-layout-step-1"/>. Furthermore,
+when performing <specref ref="semantics-style-resolution-processing-css"/>, step (4), [relative value resolution],
+the value <code>normal</code> is considered to be <emph>relative</emph> only on the most nested elements to which this property applies, namely,
```

Comment:
I don't understand the function of the words "most nested" here. Can we drop them?